build.zig

```diff
@@ -46,7 +46,11 @@ pub fn build(b: *Build) void {
     const curl_dep = b.dependency("curl", dep_opts);
     const curl_mod = curl_dep.module("curl");
 
-    const rocksdb_dep = b.dependency("rocksdb", dep_opts);
+    const rocksdb_dep = b.dependency("rocksdb", .{
```

Comment:
I think this is fine, if building a dependency in a different release mode like this won't also lead to some other issues?